angular-mentoring/src/app/coursesPage/courses/courses.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, EventEmitter, OnInit, OnChanges, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'courses',
+  templateUrl: './courses.component.html',
+  styleUrls: ['./courses.component.css']
+})
+export class CoursesComponent implements OnInit {
+  @Input() courses: Array<Object>;
+  @Output() deleteCourse = new EventEmitter<any>();
+  constructor() { }
+
+  ngOnInit() { }
+
+  ngOnChanges(){ }
```

Comment:
This class is missing inheritance from `OnChanges` interface

Reply:
I added implementing OnChanges interface.